pkg/bindings/images/images.go

```diff
@@ -310,9 +310,9 @@ func Push(ctx context.Context, source string, destination string, options entiti
 	params := url.Values{}
 	params.Set("credentials", options.Credentials)
 	params.Set("destination", destination)
-	if options.TLSVerify != types.OptionalBoolUndefined {
-		val := bool(options.TLSVerify == types.OptionalBoolTrue)
-		params.Set("tlsVerify", strconv.FormatBool(val))
+	if options.SkipTLSVerify != types.OptionalBoolUndefined {
+		val := bool(options.SkipTLSVerify == types.OptionalBoolTrue)
```

Comment:
This confused me, I think it would be better to do 

val := bool(options.SkipTLSVerify != types.OptionalBoolTrue)
params.Set("tlsVerify", strconv.FormatBool(val))

In my small brain, this is more understandable.

Reply:
I understand why your brain parser stopped here. I simplified it a bit further:
```diff
-               val := bool(options.SkipTLSVerify == types.OptionalBoolTrue)
-               params.Set("tlsVerify", strconv.FormatBool(!val))
+               // Note: we have to verify if skipped is false.
+               verifyTLS := bool(options.SkipTLSVerify == types.OptionalBoolFalse)
+               params.Set("tlsVerify", strconv.FormatBool(verifyTLS))
```